docs/class-definitions.md

```diff
@@ -544,3 +546,155 @@ Represent data that can be associated with a cursor update.
 ```ts
 type CursorData = Record<string, unknown>;
 ```
+
+## Component Locking
+
+Provides a mechanism to "lock" a component, reducing the chances of conflict in an application whilst being edited by multiple members. Inherits from [EventEmitter](/docs/usage.md#event-emitters).
+
+### Methods
+
+#### acquire
+
+Send a request to acquire a lock. Returns a Promise which resolves once the request has been sent. A successful response returns a `pending` [Lock](#lock). An error will be thrown if a lock request with a status of `pending` or `locked` already exists.
```

Comment:
I think "a successful response" is confusing copy here. It might suggest that the response is a success wrt to acquiring the lock, but it's not quite that.

Reply:
I can use here Promise nomenclature, and say the promise will resolve to a `pending` lock. I've only used successful response as that will be accurate with the authoritative server. I understand that can be confusing, what would you suggest?